checker/check_result.go

```diff
@@ -50,6 +51,10 @@ const (
 	DetailDebug
 )
 
+// ErrSuccessTotal indicates a runtime error because number of success cases should
+// be smaller than the total cases to create a proportional score.
+var ErrSuccessTotal = errors.New("unexpected number of success is higher than total")
```

Comment:
could probably be unexported, thoughts?

Reply:
How do I make it unexported? Should I declare it inside CreateProportionalScoreWeighted function?

Reply:
rename it to `errSuccessTotal`

Reply:
Learned. I didn't know about that.